tests/forward_index_test.cpp

```diff
@@ -183,6 +183,28 @@ go_bandit([]() {
             line_cfg->insert("uninvert", true);
             ceeaus_forward_test(*line_cfg);
         });
+
+        it("should analyze a new document with the current analyzer", [&]() {
+            auto cfg = tests::create_config("line");
+            auto idx = index::make_index<index::forward_index>(*cfg);
+            std::string text{"I think smoking smoking bad."};
+            corpus::document doc;
+            doc.content(text);
+            auto fvector = idx->tokenize(doc);
+
+            auto begin_sent = idx->get_term_id("<s>");
+            auto end_sent = idx->get_term_id("</s>");
+            auto bad = idx->get_term_id("bad");
+            auto smoke = idx->get_term_id("smoke");
+            auto think = idx->get_term_id("think");
+
+            AssertThat(fvector.at(begin_sent), Equals(1));
+            AssertThat(fvector.at(end_sent), Equals(1));
+            AssertThat(fvector.at(bad), Equals(1));
+            AssertThat(fvector.at(smoke), Equals(2));
+            AssertThat(fvector.at(think), Equals(1));
+            AssertThat(fvector.at(term_id{2}), Equals(0));
```

Comment:
This one, too. 2 could end up being one of the term ids actually in the sentence, especially if (when?) we switch the vocabulary map implementation.